### DISCUSSION TITLE
is sveltekit compatible with bun???

### DISCUSSION THREAD

**Discussion by H1Mohamed** [sveltejs/kit#11060] [Q&A]
I've been looking towards a lot of fast tech recently and Bun got my interest, is svelte kit fully compatible with bun???
and what happens to api routes in bun, do they use the `Bun.serve` command so maybe that I can get bun's api speed

**Comment by theArianit** [comment]
I want to know the same. I have read some reddit comments where people said they used sveltekit with bun without any major issues.

**Reply by H1Mohamed** [reply]
well if I use the api routes in sveltekit, am I still getting the performance of bun's apis???

**Comment by david-plugge** [comment]
Make sure to set up the bun adapter in your svelte.config.js and you are good to go.

**Comment by acoyfellow** [comment]
Here is a simple hello world / boilerplate using SvelteKit + Bun:
> https://gitlab.com/coeyman/sveltekit-bun-cloudrun-boilerplate
